subprojects/language-java/src/main/java/org/gradle/external/javadoc/StandardJavadocDocletOptions.java

```diff
@@ -681,6 +712,7 @@ public StandardJavadocDocletOptions noNavBar() {
      */
     private final JavadocOptionFileOption<File> helpFile;
 
+    @Optional @PathSensitive(NONE) @InputFile
```

Comment:
Good point. I checked, and the file is copied under the output directory, but it indeed retains its name. I thought it would be renamed to `help-doc.html`. The same is true for the stylesheet file.